### DISCUSSION TITLE
overset mesh solvers in blueCFD

### DISCUSSION THREAD

**Discussion by ananddoss2004** [blueCFD/Core#170] [Q&A] [answered]
Does blueCFD supports Overset meshes because i do not see those solvers in any tutorial file ?

**Comment by wyldckat** [comment, marked as answer]
Quick answer: Nope. See the answer on a recent question: https://github.com/blueCFD/Core/discussions/165

Quick note: I answered to your previous question and later moved it to the new Q&A discussions section that Github has provided us: https://github.com/blueCFD/Core/discussions/categories/q-a - so please ask this kind of future questions in that forum.
